### PR TITLE
Update twocanoes xcreds to Xcreds release 2.2

### PR DESCRIPTION
**Pull request by zacharyfleck**
Adds latest Xcreds release (2.2 - https://github.com/twocanoes/xcreds/releases/tag/realease_v2_2) manifest.

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 1]
```diff
@@ -489,7 +489,7 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>idpHostName</string>
 			<key>pfm_title</key>
-			<string>IDP Host Name</string>
+			<string>idpHostName</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: Did you mean to change the title to match the key name? Remember, `pfm_title` is the user-friendly name of the property.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 2]
```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_name</key>
 			<string>passwordElementID</string>
 			<key>pfm_title</key>
-			<string>Password Element ID</string>
+			<string>passwordElementID</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: Same as in the comment about `idpHostName`.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 3]
```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
...
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>given_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for First Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfirstname</string>
+			<key>pfm_name</key>
+			<string>map_firstname</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:535]: Unless I'm missing something, I think this should be `oidcmappingfirstname`.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 4]
```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
...
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>family_name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Last Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappinglastname</string>
+			<key>pfm_name</key>
+			<string>map_lastname</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:553]: Same as above, this should probably be `oidcmappinglastname`.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 5]
```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
...
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappingfullname</string>
+			<key>pfm_name</key>
+			<string>map_fullname</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:571]: Here too.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 6]
```diff
@@ -520,7 +520,71 @@ A profile can consist of payloads with different version numbers. For example, c
...
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>name</string>
+			<key>pfm_description</key>
+			<string>Local DS to OIDC Mapping for Name</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#oidcmappinglusername</string>
+			<key>pfm_name</key>
+			<string>map_username</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:589]: One last. This issue affects all of your new keys so maybe I'm getting it wrong and if so I'm happy to be corrected.

**Review by relgit — CHANGES_REQUESTED** [review]
Thanks for the submission, @zacharyfleck.

I've commented on a number of issues that need addressing throughout the markup, and here are a number of additional points:

- Please update `pfm_last_modified`.
- Please update `pfm_version`.
- Consider adding `pfm_app_version` to the new keys if they were added to the app recently.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 3 above)
Comment (zacharyfleck) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:535]: Regarding all the comments that had 'oidcmapping' I do believe the ones I have here are correct, but I'll confirm with TwoCanoes. I think those docs are outdated and may have been missed in their updates.

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 4 above)
Comment (zacharyfleck) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:553]: See https://github.com/DigiDNA/ProfileManifests/pull/2#discussion_r1117045104

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 5 above)
Comment (zacharyfleck) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:571]: See https://github.com/DigiDNA/ProfileManifests/pull/2#discussion_r1117045104

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 6 above)
Comment (zacharyfleck) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:589]: See https://github.com/DigiDNA/ProfileManifests/pull/2#discussion_r1117045104

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist (hunk 1 above)
Comment (zacharyfleck) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: Easy fix, I'll get it pushed up in a little bit here. This is definitely a copy and paste issue on my part.

**Comment by zacharyfleck** [comment]
@relgit Tim from Xcreds confirmed that the docs were outdated, and my keys here were correct. He is updating his documentation. I have made your requested changes.

**Comment by zacharyfleck** [comment]
You can reference fixes for OIDC docs here: https://github.com/twocanoes/xcreds/wiki/AdminGuide#map_firstname

**Review by relgit — APPROVED** [review]
Thanks for making the requested changes.

I seem to have given you `pfm_app_version` incorrectly when I should have actually given `pfm_app_min`. Sorry about that, but no worries though – I fixed it myself.

You have my thumbs up. @apizz – ready to merge?

**Pull request merged**